ui/apps/platform/cypress/integration/clusters/clustersManager.test.js

```diff
@@ -23,9 +24,15 @@ describe('Cluster managedBy', () => {
     });
 });
 
-describe('Cluster configuration', () => {
+describe('Cluster configuration classic', () => {
```

Comment:
nit: We use the term "Legacy" for the file names vs. "Classic" here. "Legacy" seems more intuitive to me vs. "Classic" which seems more like a term to describe something remarkably typical. And it would be consistent with the file names.